### PR TITLE
Create Django.gitignore

### PR DESCRIPTION
**Pull request by sarveshggn**
Added a gitignore file for Django.
I added this file because there was no gitignore file for Django.

### CODE REVIEW

**Comment by sarveshggn** [comment]
Added comments and appropriate explanation for the files.

File: Django.gitignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# You should add this if you want to deploy your site and do not want to disclose 
+# information listed above to the whole internet which can be vulnerable to security of your site
+local_settings.py
+# Databases files created when you apply migrations (after creating a model)
+db.sqlite3
+db.sqlite3-journal
+# These are the files uploaded by user on your site (eg. images, profile_picture, pdf, doc files etc.)
+/media
+# These are the static files added by admin (eg. css, js, images, pdf etc.)
+/static
```
Comment (JubayerJoy) [inline Django.gitignore:22]: Seems good to go. 👍

**Comment by JubayerJoy** [comment]
There was already a [PR](https://github.com/github/gitignore/pull/21) for Django which was merged, but can't find it.
I wonder what happened 😕

**Comment by sarveshggn** [comment]
Yes but I think it was removed, as I didn't find it there.

**Comment by 0417yjy** [comment]
I was looking for the Django gitignore. I hope this request would be approved.

**Comment by sarveshggn** [comment]
Let's hope for the best!

**Review by 53jk1 — APPROVED** [review]

**Comment by sarveshggn** [comment]
@shiftkey Can you review this pr once?

**Comment by bdougie** [comment]
Thanks, @sarveshggn, but this context already exists in https://github.com/github/gitignore/blob/master/Python.gitignore#L58

**Pull request closed without merging**